### DISCUSSION TITLE
Multiple selection

### DISCUSSION THREAD

**Discussion by radonichn** [apexcharts/apexcharts.js#2782] [Q&A]
Is there a way to do a multiple selection in pie charts:?

**Comment by jasperlg** [comment]
+1, but for all chart types. Is this something that could be added?

**Comment by fajilshaikh** [comment]
i have same question. can possible multiple selection in line charts:?
like this.
![img_20220627](https://user-images.githubusercontent.com/76145754/175913675-d22267ac-e648-439b-9dd3-dd01b145d167.png)
